README.md

```diff
@@ -116,10 +116,14 @@ make container
    ```
    $ helm install harbor-scanner-aqua ./helm/harbor-scanner-aqua \
                   --namespace harbor \
+                  --set aqua.version=4.5 \
+                  --set aqua.registry.server=aquasec.azurecr.io \
```

Comment:
Should this also use the registry.aquasec.com URL?

Reply:
Yes. We should only use `aquasec.azurecr.io` during development.